static/scss/email-composition-dialog.scss

```diff
@@ -1,67 +1,125 @@
-        font-size: 16px;
-        font-weight: normal;
+      .subheading-section {
+        padding-bottom: 20px;
 
-        &.default {
-          font-style: italic;
+        .subheading-to {
+          padding: 0 10px;
         }
 
-        &.rounded {
-          background-color: $bg-med-gray;
-          line-height: 22px;
-          padding: 10px 10px 10px 20px;
-          border-radius: 20px;
-          font-style: normal;
-        }
+        .subheading {
+          letter-spacing: normal;
+          margin: 0;
+          padding: 0;
+          font-size: 16px;
+          font-weight: normal;
 
-        &.profile-image-bubble {
-          background-color: $bg-med-gray;
-          border-radius: 50px;
-          padding: 0 30px 0 0;
-          height: 43px;
-          display: inline-block;
+          &.default {
+            font-style: italic;
+          }
+
+          &.rounded {
+            background-color: $bg-med-gray;
+            line-height: 22px;
+            padding: 10px 10px 10px 20px;
+            border-radius: 20px;
+            font-style: normal;
+          }
 
-          >span {
+          &.profile-image-bubble {
+            background-color: $bg-med-gray;
+            border-radius: 50px;
+            padding: 0 30px 0 0;
+            height: 43px;
             display: inline-block;
-            padding-left: 15px;
+
+            > span {
+              display: inline-block;
+              padding-left: 15px;
+            }
           }
         }
       }
+
+      .validation-error {
+        font-size: 12px;
+        color: $validation-red;
+      }
+
+      textarea {
+        padding: 15px;
+        border-color: $border-color;
+        border-radius: 2px;
+        overflow: hidden;
+
+        &.email-subject {
+          padding: 8px 15px;
+          margin-bottom: 5px;
+          height: 60px;
+        }
+      }
+
+      .email-automatic {
+        padding: 20px 0 0;
+      }
     }
+  }
+}
+
+.email-type {
+  width: 100%;
+  margin: 0 0 10px;
+  background-color: $email-composition-type;
+  padding: 15px;
+  border-radius: 8px;
+
+  .type-radio-group {
+    display: flex;
+    flex-direction: row;
 
-    .validation-error {
-      font-size: 12px;
-      color: $validation-red;
+    label {
+      color: $darkergray;
+      font-size: 15px;
     }
+  }
 
-    textarea {
-      padding: 15px;
-      border-color: $border-color;
-      border-radius: 2px;
+  .one-time-email {
+    display: flex !important;
+    flex-direction: row;
+    flex-grow: 1;
+    width: 55% !important;
 
-      &.email-subject {
-        margin-bottom: 5px;
-      }
+    > div > div {
+      margin-right: 6px !important;
     }
+  }
 
-    .email-automatic {
-      padding: 20px 0;
+  .email-campaign {
+    display: flex !important;
+    flex-direction: row;
+    flex-grow: 5;
+
+    > div > div {
+      margin-right: 6px !important;
     }
   }
+
+  .email-campaign-content {
+    margin: 10px 5px 0;
+    font-size: 14px;
+  }
 }
```

Comment:
the `find` method provided by Enzyme will return all items that match, so you shouldn't need the classnames to get the `input` elements.